stonybook/pipeline/episode_break_prominence/add_break_prominence.py

```diff
@@ -0,0 +1,189 @@
+import os
+import lxml.etree as ET
+import pickle
+from nltk.corpus import stopwords
```

Comment:
Instead of using nltk, we should use the stopwords from the spaCy library as that will be a dependency anyways.